pkg/csi/service/osutils/linux_os_utils_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build darwin || linux
+// +build darwin linux
+
+package osutils
+
+import (
+	"strconv"
+	"testing"
+)
+
+func TestUnescape(t *testing.T) {
+	tests := []struct {
+		in, out     string
+		expectError bool
+	}{
+		{
+			// Space is unescaped. This is basically the only test that can happen in reality.
```

Comment:
You have the following in the PR description. Can you add that here as well?
`This happens only for in-line in-tree vSphere volumes when CSI migration is enabled.`

Reply:
Fixed

Reply:
Did you add this comment somewhere?  I don't see it.

Reply:
@jsafrane Did you add the comment somewhere?  I can't find it.

Reply:
Oops, sorry, I forgot to push